common/app/model/meta.scala

```diff
@@ -305,7 +305,7 @@ object ContentFormat {
   implicit val contentFormatReads = contentFormatBuilder.apply(ContentFormat.apply _)
 }
 
-final case class MetaData(
+case class MetaData(
```

Comment:
Shouldn't this remain here to avoid us extending it?

Reply:
If we make this class final I can't mock it in the tests, so it was a compromise. I don't think anyone would try to extend this class (why would you?!) but I can add a comment to make this clearer if you think it's necessary?